mlir/test/python/pass_manager.py

```diff
@@ -281,3 +281,26 @@ def testPostPassOpInvalidation():
         # CHECK:   return
         # CHECK: }
         log(module)
+
+
+# Test -print-ir-after-all.
```

Comment:
This is for human consumption I guess? Like info on what the test accomplishes? Not sure if it's necessary (pretty self-evident). Also a little confusing...

Reply:
I added that because some other tests also had it, but you're absolutely right. It's doesn't say much (apart from the `-print-ir-after-all` mention. I've removed it now